### PR TITLE
Added table of open ports to installation guide. (BSC#1043784)

### PR DESCRIPTION
**Pull request by lproven**
# Table of ports

Added a table with a list of open ports to the Networking section of the deployment guide. 

Restructured the table from previous branch:

* merged header fields across multiple rows
* replaced empty entries with <entry/> tag
* fixed indentation
* modified XML ID to conform to deployment guide convention

### CODE REVIEW

File: xml/deployment_installation.xml [hunk 1]
```diff
@@ -83,7 +83,239 @@
...
+          </entry>
+          <entry>
+           <para> Yes </para>
+          </entry>
+          <entry>
+           <para>
+            <literal>etcd</literal> control </para>
+          </entry>
+          <entry>
+           <para> For peer-to-peer etcd traffic </para>
```
Comment (cjschroder) [inline xml/deployment_installation.xml]: Is there a reason that etcd does not always have <literal> tags?

Comment (cjschroder) [reply xml/deployment_installation.xml]: Line 63: "all nodes should to be"

Comment (cjschroder) [reply xml/deployment_installation.xml]: I'm probably using github's review dealie wrong-- line 63 "all nodes should to be"

Comment (lproven) [reply xml/deployment_installation.xml]: I did not tag ``etcd'' in the Notes column, but it *is* inconsistent. Fixed.

"All nodes should be on the same network" - fixed.

Comment (cjschroder) [reply xml/deployment_installation.xml]: Line 222-- etcd again :)

Comment (cjschroder) [reply xml/deployment_installation.xml]: I'm not sure that replaceable is the right tag for this-- maybe check with the team tomorrow. Maybe it should be the command tag.

**Comment by lproven** [comment]
On Wed, 24 Jan 2018 18:29:58 +0000 (UTC)
Carla Schroder <notifications@github.com> wrote:

> I'm not sure that replaceable is the right tag for this-- maybe check with the team tomorrow. Maybe it should be the command tag.

I haven't used the <replaceable> tag anywhere, have I?

-- 
Liam Proven - Technical Writer, SUSE Linux s.r.o.
Corso II, Křižíkova 148/34, 186-00 Praha 8 - Karlín, Czechia
Email: lproven@suse.com - Office telephone: +420 284 241 084

File: xml/deployment_installation.xml (hunk 1 above)
Comment (lproven) [reply xml/deployment_installation.xml]: I haven't used the <replaceable> tag anywhere, have I?

**Comment by cwickert** [comment]
@lproven, here it makes sense to have two separate commits: One for the table, the other for the changes @cjschroder requested.

I've manually rebased the branch and 'fixed up' the the last commit so it becomes part of the second one. This is what I'll merge now.

**Pull request merged**